gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -788,7 +788,7 @@ gst_tensor_filter_common_set_property (GstTensorFilterPrivate * priv,
       if (fw) {
         /** Get framework info for v1 */
         if (GST_TF_FW_V1 (fw) &&
-            priv->fw->getFrameworkInfo (prop, NULL, &priv->info) < 0) {
+            fw->getFrameworkInfo (prop, NULL, &priv->info) < 0) {
```

Comment:
This is a question to improve my understanding.
Is there a case where the potive value is returned? 
The description only says '0 if OK. non-zero if error.'

Reply:
According to the description, positive value == undefined behavior.
This is a callback defined by filter subplugins; thus, we cannot guarantee it.

Note that this is a common practice in C libraries.

Anyway, in the future, we may generate fatal errors if it returns > 0 so that subplugin writers may fix it right away. You will need some survey (on well-known and well-written C programs) to determine if we need to generate such or not.